README.md

```diff
@@ -1 +1,192 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- - Not available
+
+## 🚀 Live Demo <a name="live-demo"></a> -->
+
+<!-- - [Live Demo Link](<replace-with-your-deployment-URL>)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Laptop
+- Github
+- Git
+- Ruby
+- Linters
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install [Ruby](https://www.ruby-lang.org/en/documentation/installation/).
+- Create a repo on your repositores files.
+- Clone or make a copy of this repo on your local machine.
+- Follow GitHub flow.
+- A carefully reading of this README.md is required.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/kerlos-bahaa/TDD-project.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+
+### Usage
+
+To run the project, run the following command:
+
+```sh
+  ruby main.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rubocop
+rspec spec
+```
+
+<!-- ### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Kerlos bahaa**
+
+- GitHub: [Kerlos bahaa](https://github.com/kerlos-bahaa)
+- LinkedIn: [Kerlos bahaa
+  ](https://www.linkedin.com/in/kerlos-bahaa/)
+
+👤 **Sana Tameem**
+
+- GitHub: [@githubhandle](https://github.com/SanaTameem)
+- Twitter: [@twitterhandle](https://twitter.com/sana_tameem)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sana-tameem/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Performance Optimization
+- Improved Error Handling
+- Customizable FizzBuzz Rules
+- Additional Math Functions
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kerlos-bahaa/TDD-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project,please give us a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me the opportunity to learn and grow as a developer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/kerlos-bahaa/TDD-project/blob/TDD/MIT.md) licensed.
```

Comment:
- [ ] Date(year) in the license file should be edit to match the currently year(2023). This will render your readme file professional.